librocksdb_sys/crocksdb/crocksdb/c.h

```diff
@@ -940,10 +943,10 @@ extern C_ROCKSDB_LIBRARY_API void crocksdb_options_enable_statistics(
     crocksdb_options_t*, unsigned char);
 extern C_ROCKSDB_LIBRARY_API void crocksdb_options_reset_statistics(
     crocksdb_options_t*);
-extern C_ROCKSDB_LIBRARY_API bool crocksdb_load_latest_options(
+extern C_ROCKSDB_LIBRARY_API unsigned char crocksdb_load_latest_options(
```

Comment:
`bindgen` generates FFI based on `c.h`, but why we modify `c.h`?

Reply:
@Connor1996 this function was added by us, so not conform to c's style.

Reply:
rocksdb's c.h has convention: 
`Bools have the type unsigned char (0 == false; rest == true)`

Reply:
got it

Reply:
can you extract them to a separate PR?

Reply:
OK